src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -177,7 +177,7 @@ namespace JSONHelpers
         }
     }
 
-    bool FancyZonesData::GetAppLastZone(HWND window, PCWSTR appPath, _Out_ PINT iZoneIndex) const
+    int FancyZonesData::GetAppLastZone(HWND window, PCWSTR appPath, _Out_ PINT iZoneIndex) const
```

Comment:
`iZoneIndex` as an argument is unnecessary now. It's confusing since you change it and return same value.

Reply:
Will change